webhooks/ssp_webhook_test.go

```diff
@@ -239,17 +248,20 @@ var _ = Describe("SSP Validation", func() {
 
 		It("should reject URL without https:// or ssh://", func() {
 			sspObj.Spec.CommonInstancetypes.URL = pointer.String("file://foo/bar")
-			Expect(validator.ValidateCreate(ctx, toUnstructured(sspObj))).ShouldNot(Succeed())
+			_, err := validator.ValidateCreate(ctx, toUnstructured(sspObj))
+			Expect(err).ShouldNot(Succeed())
```

Comment:
I'd go for one style throughout the whole file, either `HaveOccured()` or `Succeed()`.

Reply:
done